metric/system/process/helpers.go

```diff
@@ -99,6 +99,33 @@ func GetProcCPUPercentage(s0, s1 ProcState) ProcState {
 
 }
 
+// NonFatalErr is returned when there was an error
+// collecting metrics, however the metrics already
+// gathered and returned are still valid.
+// This error can be safely ignored, this will result
+// in having partial metrics for a process rather than
+// no metrics at all.
+//
+// It was introduced to allow for partial metrics collection
+// on privileged process on Windows.
```

Comment:
[Suggestion]
It feels a bit out of place a type state when it's returned, therefore I'd suggest:

```suggestion
// NonFatalErr indicates an error occurred during metrics
// collection, however the metrics already
// gathered and returned are still valid.
// This error can be safely ignored, this will result
// in having partial metrics for a process rather than
// no metrics at all.
//
// It was introduced to allow for partial metrics collection
// on privileged process on Windows.
```

Reply:
Thank!